lib/kino_flame/runner_cell.ex

```diff
@@ -5,13 +5,33 @@ defmodule KinoFLAME.RunnerCell do
   use Kino.JS.Live
   use Kino.SmartCell, name: "FLAME runner cell"
 
-  @text_fields ["name"]
+  @text_fields ["name", "backend"]
   @number_fields ["min", "max", "max_concurrency"] ++ ["fly_cpus", "fly_memory_gb", "fly_gpus"]
+  @default_pod_template """
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    generateName: livebook-flame-runner-
+  spec:
+    containers:
+      - image: ghcr.io/livebook-dev/livebook:nightly
```

Comment:
Hmm, if livebook is running on latest, this should be latest. Is there an env var or something we could check to see which image the current pod is running?

Reply:
Oooh actually, the image is set by `flame_k8s_backend` anyway. I should remove it here!